src/main/java/tutu/user/Parser.java

```diff
@@ -0,0 +1,32 @@
+package tutu.user;
+
+import tutu.exception.InvalidInputException;
+
+public class Parser {
+
+    public enum Function {
+        LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE, FIND
+    }
```

Comment:
I like that you included enum into your code 👍

Reply:
I agree with @atmh, using enum makes the code more clean and concise. 👍